leo_bringup/scripts/leo_system

```diff
@@ -17,11 +17,20 @@ def shutdownCallback(x):
     os.system("shutdown -h now")
 
 
+def publishNamespace():
+    namespace_pub = rospy.Publisher("ros_namespace", String, queue_size=1, latch=True)
+    namespace = rospy.get_namespace()
+    msg = "" if namespace == None else str(namespace)
```

Comment:
Isn't this line redundant when using rospy.get_namespace() ?

Reply:
Yes, it was needed when using os.getenv(). Removing it now.